src/__tests__/fixtures/bundle-bug-9567.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
could we shrink down the sample and make it fit into existing suite?

Reply:
This is already shrunk a lot :D

Reply:
perhaps, but we certainly don't need it to be that deeply nested, and properties like "title" etc. could all be deleted.

Reply:
I spent over an hour shrinking it down already 😆

Reply:
Shrink it yourself.

Reply:
done